packages/safe-tools-client/app/components/schedule-payment-form-action-card/ui/index.gts

```diff
@@ -295,51 +297,66 @@ export default class SchedulePaymentFormActionCardUI extends Component<Signature
           </div>
         </BoxelField>
       </Section>
-        <ActionChin @state={{this.actionChinState}}>
-          <:default as |ac|>
+      <ActionChin @state={{this.actionChinState}}>
+        <:default as |ac|>
+          {{#if @isSafesEmpty}}
```

Comment:
If safes are empty, then display the Schedule Payment button? Isn't this the other way around as it should be?